components/BlogCard.tsx

```diff
@@ -2,7 +2,13 @@ import Link from "next/link";
 import BlurImage from "./BlurImage";
 import Date from "./Date";
 
-export default function BlogCard({ data }) {
+import type { PostData } from "@/types";
+
+interface BlogCardProps {
+  data: PostData;
+}
```

Comment:
What do you think about instead of using `interface`  use the normal `type alias`?
It's a bit less verbose, it doesn't mix with the idea of OOP (which is common to Classes, Interfaces, and extends), it's easier to use `unions`, and it won't have the [issue/risk of merging two interfaces with the same name](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces)

```suggestion
type BlogCardProps = {
  data: PostData;
}
```

Reply:
Our style guide prefers imports. There's some discussion as to why [in this thread](https://vercel.slack.com/archives/C02HH79U277/p1642180308007800).

Reply:
Uh, that's very interesting, will have a look, thanks! =D